### DISCUSSION TITLE
Misskeyインスタンスでファイアウォールは必要ですか？ | Is firewall needed in Misskey instances?

### DISCUSSION THREAD

**Discussion by Lefioty** [misskey-dev/misskey#13720] [Q&A] [answered]
[日本語]
Misskeyサーバーをホストする際に、ufwの使用をお勧めしますか？
その場合、どのポートを許可する必要がありますか？

可能なポートについて調べました：
- sshd
- https 443
- PostgreSQL 5432
- Redis 6379
- Docker Compose? 3000

ちなみに、私のインスタンスではCloudflareプロキシが有効になっています。

---
[English]
May I ask is ufw recommended while hosting Misskey server?
If so, what ports should be allowed? 

I did some research which ports may be needed:
- sshd
- https 443
- PostgreSQL 5432
- Redis 6379
-  Docker Compose? 3000

BTW Cloudflare proxy is enabled in my instances.

**Comment by Hoto-Cocoa** [comment, marked as answer]
You should open SSH(TCP/22) port for you. This not required to users.

You should NOT open PostgreSQL/Redis/Docker Compose ports.

Required ports: TCP/80, TCP/443, UDP/443(some situations)

**Reply by Lefioty** [reply]
Got it! Thanks 😃
